src/core/Object3D.js

```diff
@@ -31,20 +31,24 @@ function Object3D() {
 
 	this.up = Object3D.DefaultUp.clone();
 
-	var position = new Vector3();
+	var position = new Vector3WithDirtyFlag();
 	var rotation = new Euler();
 	var quaternion = new Quaternion();
```

Comment:
I don't make `QuaternionWithDirtyFlag` yet in this PR because not sure why but somehow using the extension of `Quaternion` seems to lead the bad performance on my Windows Chrome...